app/src/main/java/com/capstone/weatherapp/MainActivity.kt

```diff
@@ -1,11 +1,22 @@
 package com.capstone.weatherapp
 
 import android.os.Bundle
-import androidx.activity.ComponentActivity
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 
-class MainActivity : ComponentActivity() {
+class MainActivity : AppCompatActivity() {
+    lateinit var cityList : ArrayList<City>
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.fragment_city_list)
```

Comment:
Instead of setting the content view to a fragment, let's add `viewBinding` and use the `ActivityMainBinding` to inflate the layout